auth-source-pass.el

```diff
@@ -80,6 +80,9 @@ See `auth-source-search' for details on SPEC."
   ;; clear the cache (required after each change to #'auth-source-pass-search)
   (auth-source-forget-all-cached))
 
+(defvar auth-source-pass-path "~/.password-store"
```

Comment:
could you please move that to the beginning of the file, right after the `require` lines?

Reply:
please use `defcustom` instead of `defvar`. You will have to create a `defgroup` first.